doc/nrf/ug_multi_image.rst

```diff
@@ -73,14 +82,14 @@ When building the parent image, you can configure how the child image should be
 
 * Build the child image from the source and include it with the parent image.
   This is the default setting.
-* Use a prebuilt HEX file of the child image and include it with the parent image.
+* Use a prebuilt hexidecimal file of the child image and include it with the parent image.
```

Comment:
In this case, I think "HEX file", "hex file" or "Intel hex file" is appropriate, as it's a valid concept on its own, and no longer an abbreviation: https://en.wikipedia.org/wiki/Intel_HEX